src/components/ErrorField/ErrorField.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (error: string) => void;
+};
+
+export const ErrorField = ({ errorMessage, setErrorMessage }: Props) => {
+  useEffect(() => {
+    setTimeout(() => setErrorMessage(''), 3000);
```

Comment:
You should do it only in case of error, and a good practice to remove timeouts/timers in case of component unmount